src/util/artifacts.js

```diff
@@ -12,7 +12,8 @@ export async function prepareArtifactsDir(
   try {
     const stats = await fs.stat(artifactsDir);
     if (!stats.isDirectory()) {
-      throw new WebExtError(`${artifactsDir} is not a directory`);
+      throw new UsageError(
+        `artifacts-dir=${artifactsDir} (this value is not a directory)`);
```

Comment:
This should be `--artifacts-dir` (with the leading dashes) so that it's clearly a command line option